tests/integration/logging/monolog3/test_monolog_basic_clm.php

```diff
@@ -0,0 +1,205 @@
+          "hostname": "__HOST__"
+        },
+        {
+          "message": "warning",
+          "level": "WARNING",
+          "timestamp": "??",
+          "trace.id": "??",
+          "span.id": "??",
+          "entity.guid": "??",
+          "entity.name": "tests/integration/logging/monolog3__FILE__",
+          "hostname": "__HOST__"
+        },
+        {
+          "message": "info",
+          "level": "INFO",
+          "timestamp": "??",
+          "trace.id": "??",
+          "span.id": "??",
+          "entity.guid": "??",
+          "entity.name": "tests/integration/logging/monolog3__FILE__",
+          "hostname": "__HOST__"
+        },
+        {
+          "message": "alert",
+          "level": "ALERT",
+          "timestamp": "??",
+          "trace.id": "??",
+          "span.id": "??",
+          "entity.guid": "??",
+          "entity.name": "tests/integration/logging/monolog3__FILE__",
+          "hostname": "__HOST__"
+        },  
+        {
+          "message": "emergency",
+          "level": "EMERGENCY",
+          "timestamp": "??",
+          "trace.id": "??",
+          "span.id": "??",
+          "entity.guid": "??",
+          "entity.name": "tests/integration/logging/monolog3__FILE__",
+          "hostname": "__HOST__"
+        },        
+        {
+          "message": "debug",
+          "level": "DEBUG",
+          "timestamp": "??",
+          "trace.id": "??",
+          "span.id": "??",
+          "entity.guid": "??",
+          "entity.name": "tests/integration/logging/monolog3__FILE__",
+          "hostname": "__HOST__"
+        }
+      ]
+    }
+  ]
+ */
+
+require_once(realpath(dirname(__FILE__)) . '/../../../include/config.php');
+require_once(realpath(dirname(__FILE__)) . '/../../../include/monolog.php');
+require_monolog(3);
+
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use Monolog\Formatter\LineFormatter;
+
+
+function test_logging() {
+    $logger = new Logger('monolog3');
+
+    $logfmt = "%channel%.%level_name%: %message% %context%\n";
+    $formatter = new LineFormatter($logfmt);
+
+    $stdoutHandler = new StreamHandler('php://stdout', Logger::DEBUG);
+    $stdoutHandler->setFormatter($formatter);
+
+    $logger->pushHandler($stdoutHandler);
+    
+    // insert delays between log messages to allow priority sampling
+    // to resolve that later messages have higher precedence
+    // since timestamps are only millisecond resolution
+    // without delays sometimes order in output will reflect
+    // all having the same timestamp.
+    $logger->debug("debug");
+    usleep(10000);
+    $logger->info("info");
+    usleep(10000);
+    $logger->notice("notice");
+    usleep(10000);
+    $logger->warning("warning");
+    usleep(10000);
+    $logger->error("error");
+    usleep(10000);
+    $logger->critical("critical");
+    usleep(10000);
+    $logger->alert("alert");
+    usleep(10000);
+    $logger->emergency("emergency");
+}
+
```

Comment:
I wonder how would the EXPECT_SPAN_EVENTS look like if a custom tracer `newrelic_add_custom_tracer("test_logging");` was added here...
```suggestion
newrelic_add_custom_tracer("test_logging");
```

Reply:
custom tracing is already handled in 4 other CLM tests.  In this case even though we log it, it would be no different from another custom_tracer. 

Many of the tests will need updating once CLM is on by default.

Reply:
Thank you! This is exactly what I wanted to see tested: CLM attributes in spans for auto-instrumented functions.